native/acter/src/api/common.rs

```diff
@@ -69,6 +69,29 @@ impl OptionRsvpStatus {
     pub fn status(&self) -> Option<RsvpStatus> {
         self.status.clone()
     }
+
+    pub fn status_str(&self, pascal_case: bool) -> Option<String> {
+        let status = self.status.clone();
+        if pascal_case {
+            if let Some(status) = status {
+                match status {
+                    RsvpStatus::Yes => {
+                        return Some("Yes".to_owned());
+                    }
+                    RsvpStatus::Maybe => {
+                        return Some("Maybe".to_owned());
+                    }
+                    RsvpStatus::No => {
+                        return Some("No".to_owned());
+                    }
+                }
+            }
+            None
```

Comment:
this could still be a much briefer map ...
```suggestion
            status.map(|status| => 
                match status {
                    RsvpStatus::Yes => "Yes",
                    RsvpStatus::Maybe => "Maybe",
                    RsvpStatus::No => "No",
                }.to_owned())
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1401/commits/9b29ff37073e4a5b50062bd1c4b1792a9e0686f5